### PR TITLE
Add reference Apline base image and add screencast for private image

### PR DESCRIPTION
**Pull request by axonasif**
## Related Issue(s)

Compliments https://linear.app/gitpod/issue/COM-6/using-custom-docker-images

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

-   [x] Test all links to make sure they are working correctly
-   [x] Verify that all forms (sales, support, docs feedback) are functioning properly and data is being captured correctly
-   [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
-   [x] Ensure that the website is mobile responsive and functions appropriately on different devices
-   [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

-   [x] Check all website copies for accuracy and spelling/grammar errors
-   [x] Verify that all images and videos are displaying properly
-   [x] Ensure that all downloadable files are working correctly
-   [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

-   [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
-   [x] Test website load times to ensure that the website is loading quickly
-   [x] Verify that the website can handle high traffic volumes without crashing
-   [x] Check that all website functionality is working properly under heavy load
-   [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

-   [x] Ensure that the website meets basic accessibility guidelines.
-   [x] Test that users with disabilities can easily navigate and use the website
-   [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

-   [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
-   [x] Ensure that all sensitive data is being stored and transmitted securely

</details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3906"></a>

### CODE REVIEW

File: src/content/docs/configure/workspaces/workspace-image.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ By default, Gitpod uses a standard Docker Image called [`Workspace-Full`](https:
 
 If this image does not include the tools you need for your project, you can provide a public Docker image or your own [Dockerfile](#configure-a-custom-dockerfile). This provides you with the flexibility to install the tools & libraries required for your project.
 
-> **Note:** Gitpod supports Debian/Ubuntu based Docker images. Alpine images do not include [libgcc and libstdc++](https://code.visualstudio.com/docs/remote/linux#_tips-by-linux-distribution) which breaks Visual Studio Code. See also [Issue #3356](https://github.com/gitpod-io/gitpod/issues/3356).
+> **Note:** Gitpod supports Debian/Ubuntu based Docker images out-of-the-box. Some base images like Alpine do not include [libgcc and libstdc++](https://code.visualstudio.com/docs/remote/linux#_tips-by-linux-distribution) by default which breaks Visual Studio Code. See a [reference Alpine base image](#alpine) and [issue #3356](https://github.com/gitpod-io/gitpod/issues/3356).
```
Comment (ghostdevv) [inline src/content/docs/configure/workspaces/workspace-image.md:13]:
```suggestion
> **Note:** Gitpod supports Debian/Ubuntu based Docker images out-of-the-box. Some base images like Alpine do not include [libgcc and libstdc++](https://code.visualstudio.com/docs/remote/linux#_tips-by-linux-distribution) by default, which breaks Visual Studio Code. See a [reference Alpine base image](#alpine) and [issue #3356](https://github.com/gitpod-io/gitpod/issues/3356).
```

File: src/content/docs/configure/workspaces/workspace-image.md [hunk 2]
```diff
@@ -115,7 +117,9 @@ See also [Gero's blog post](/blog/docker-in-gitpod) running through an example.
 
 While it is recommended to extend one of the <a href="https://hub.docker.com/u/gitpod/" target="_blank">Gitpod-provided base images</a> for custom Dockerfiles to ensure the image has the required dependencies for a workspace, it is possible to configure a Dockerfile with a public (Debian/Ubuntu-based) image as its base.
 
-There are some requirements though for a public base image to work properly as a workspace. See the below Dockerfile as a reference. For instance, you'll need to set up the `gitpod` user with the right UID, and install `git` to enable your configured dotfiles for the workspace.
+There are some requirements for a public base image to work properly as a workspace. See the below Dockerfile(s) as a reference. For instance, you'll need to set up the `gitpod` user with the right UID, and install `git` to ensure your workspace can start.
```
Comment (ghostdevv) [inline src/content/docs/configure/workspaces/workspace-image.md:120]:
```suggestion
There are some requirements for a public base image to work properly as a workspace. For example, you'll need to set up the `gitpod` user with the right UID, and install `git` to ensure your workspace can start. See the below Dockerfiles as a reference.
```

File: src/content/docs/configure/workspaces/workspace-image.md (hunk 1 above)
Comment (axonasif) [reply src/content/docs/configure/workspaces/workspace-image.md:13]: Thanks!

**Comment by axonasif** [comment]
Thanks for the approval @Furisto!

**Pull request merged**